### DISCUSSION TITLE
Match within subdirectories with `**/*` not working

### DISCUSSION THREAD

**Discussion by awkaiser-tr** [community/community#10603] [Other Feature Feedback, Questions, & Ideas] [answered]
Matching within subdirectories following [path qualifier documentation](https://cs.github.com/about/syntax#path):
```
path:/src/**/*.js
```
... always returns zero results. Regular expression works great:
```
path:/src\/.*\.js/
```
🍻  **I'm `hella excited` about this new Code Search, _thank you!_** 🍻

**Comment by colinwm** [comment, marked as answer]
Hey @awkaiser-tr, good find! That was a bug in our glob --> regex conversion. It's fixed now. Thanks for the tip!
